script/Arcueid/Day 12/12_00_ARC12_2.txt

```diff
@@ -76,7 +76,7 @@ Beautiful as relief carving on glass.
 [sha:276602fbe0396ac9cd82a4cbb69279aaaa0624aa]{
 -- Page 42, Offset 30610.
 -- 鮮明すぎて空からこぼれ落ちてきそうな白貌。
-A pure, white face so vivid that it feels like it'll fall from the sky.
+A pure, white visage so vivid it feels like it'll fall from the sky.
```

Comment:
"visage" twice?

Reply:
suggestions? something like "countenance" or just revert to "face"?

Reply:
I'd say keep this one visage and make the other face.